wasm/bindings/TranslationModelBindings.cpp

```diff
@@ -31,9 +35,11 @@ TranslationModel* TranslationModelFactory(const std::string &config,
 EMSCRIPTEN_BINDINGS(translation_model) {
   class_<TranslationModel>("TranslationModel")
     .constructor(&TranslationModelFactory, allow_raw_pointers())
-    .function("translate", &TranslationModel::translate)
+    .function("translate", &TranslationModel::translateMultiple)
```

Comment:
Is this api change?

Reply:
We (`Service`) have a future returning API (translate) which is expected to work with async when time comes. It's been deferred for other changes now. `TranslationModel` (Mozilla API) converted this into a blocking API, return `vector<Response>` for texts. 

Collapse thus does `TranslationModel::translate` -> `Service::translateMultiple` (which can handle multiple text -> multiple Responses). This gives them the benefits of proper batching compiling from multiple requests which didn't exist before. The signatures were different enough to create a new function in Service for the same. 

From a layman point of view, they can now translate individual elements (and not whole page) without degrading speed. This is therefore also a workaround for https://github.com/browsermt/bergamot-translator/issues/35 which came as a byproduct.